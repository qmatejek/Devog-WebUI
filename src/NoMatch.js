import React from 'react'

export const NoMatch=()=>(
    <div>
        <h2>These aren't the droids you're looking for.</h2>
    </div>
    
)