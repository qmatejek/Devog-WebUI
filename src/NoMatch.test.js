import { NoMatch } from './NoMatch';
import React from 'react';
import { shallow, render } from 'enzyme';

it("renders correctly", () => {
    const wrapper = shallow(
      <NoMatch />
    );
  
    expect(wrapper).toMatchSnapshot();
  });