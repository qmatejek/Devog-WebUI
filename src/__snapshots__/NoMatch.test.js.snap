// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2>
    These aren't the droids you're looking for.
  </h2>
</div>
`;
